qcodes/instrument/group_parameter.py

```diff
@@ -55,11 +55,9 @@ def __init__(self,
         self._initial_value = initial_value
         super().__init__(name, instrument=instrument, **kwargs)
 
-        self.set = self._wrap_set(self.set_raw)
-
-        self.get_raw = lambda result=None: result if result is not None else self._get_raw_value()  # type: ignore[assignment]
-
-        self.get = self._wrap_get(self.get_raw)
+    def get_raw(self, parameter_value: Any = ParamRawDataType) -> ParamRawDataType:
```

Comment:
```suggestion
    def get_raw(self, parameter_value: ParamRawDataType= None) -> ParamRawDataType:
```
Sorry I think I messed this one up

Reply:
I fixed it too